docs/hardware.md

```diff
@@ -473,11 +473,17 @@ VirtualBox is a general-purpose full virtualiser for x86 hardware, targeted at s
 
 The VirtualBox virtual machine is great for those occasions where SBC performance just isn't enough. Run one of these on any x86_64 PC/server and still get the same great DietPi features and experience.
 
-### Parallels Desktop
+### Parallels Desktop (macOS)
 
 ![macOS Parallels Desktop](assets/images/Parallels3.png){: width="270" height="82" loading="lazy"}
 
-The Parallels Desktop virtual machine for macOS is great for those occasions where SBC performance just isn't enough. Run one of these on any x86_64 macOS system (e.g. Mac mini 2011/2012/2014/2018) and still get the same great DietPi features and experience.
+The [Parallels Desktop virtual machine for macOS](https://www.parallels.com/products/desktop/) is great for those occasions where SBC performance just isn't enough. Run one of these on any x86_64 macOS system (e.g. Mac mini 2011/2012/2014/2018) and still get the same great DietPi features and experience.
+
+### UTM (macOS)
+
+![macOS UTM](assets/images/UTM3.png){: width="270" height="221" loading="lazy"}
+
+The [UTM virtual machine for macOS](https://mac.getutm.app/) is great for those occasions where SBC performance just isn't enough. Run one of these on any macOS system (e.g. Mac mini 2011/2012/2014/2018 and M1 Apple Silicon) and still get the same great DietPi features and experience.
```

Comment:
It is able to run `x86_64` on an M1 ARM host system? Does it somehow include emulation (on top of virtualisation) or would this require an ARM VM image? In case of the letter, let's not name M1 here, basically using the text from Parallels that "any x86_64 macOS system" works.

Reply:
I moved the chapter from the bottom to an info box at the top.